client/bin/multi-client.js

```diff
@@ -441,7 +459,23 @@ class State {
 		const balenaCloud = new BalenaCloudInteractor(balena);
 		// Iterates through test jobs and pushes jobs to available testbot workers
 		for (const runConfig of runConfigs) {
-			if (runConfig.workers instanceof Array) {
+			// If its a qemu worker, the worker url will be localhost - so for each job targetted at qemu, we can create a core/worker pairing
+			if(runConfig.workers.includes(`http://localhost`)) {
+				// find 2 unused ports, and make the node processes inside the containers we generate listen on them
+				// This is to avoid port conflicts between the containers we generate
+				let availablePorts = await fp(5000, 5100, '127.0.0.1', 2);
```

Comment:
```suggestion
				let availablePorts = await fp(5000, 6000, '127.0.0.1', 2);
```
No point in introducing risk of running out of ports if teardown is broken.